docs/en/observability/configure-uptime-settings.asciidoc

```diff
@@ -32,34 +32,34 @@ data outside of this pattern.
 image::images/heartbeat-indices.png[Heartbeat indices]
 
 [[configure-uptime-alert-connectors]]
-== Configure alert connectors
+== Configure connectors
```

Comment:
Nice. I think this works well until Uptime migrates away from the deprecated alerting API. There's an issue in Kibana somewhere...

Reply:
Thanks for letting me know.